modules/chassis_elasticsearch/manifests/init.pp

```diff
@@ -11,9 +11,6 @@
     class { 'elasticsearch':
       ensure => 'absent'
     }
-    package { 'java-common':
```

Comment:
For reference this was breaking using the `disabled_extensions` feature with this extension

Reply:
Oohh let me dig into this one and open an issue to fix it! Thanks for letting me know!